### PR TITLE
Create Collateral-Omnibus-8.md

### PR DESCRIPTION
**Pull request by ChandlerDeKock**

### CODE REVIEW

File: Collateral-Omnibus-8.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+To accomplish this upgrade, the following changes need to be added to the collateral currency whitelist introduced in UMIP-8.
+
+### AQUA
+-   The **AQUA** address [token address](https://etherscan.io/address/0xd34a24006b862f4e9936c506691539d6433ad297): 0xd34a24006b862f4e9936c506691539d6433ad297 
+-   A final fee of **700 000 AQUA** needs to be added for the Store contract.
+    
+### IDIA
+-   The **IDIA** address [token address](https://etherscan.io/address/0x0b15ddf19d47e6a86a56148fb4afffc6929bcb89): 0x0b15ddf19d47e6a86a56148fb4afffc6929bcb89 
+-   A final fee of **450 IDIA** needs to be added for the Store contract.
+
```
Comment (mhairimca) [inline Collateral-Omnibus-8.md:27]: Token contract has no liquidity on ethereum only on binance smart chain.

File: Collateral-Omnibus-8.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Rationale
+
+This store fee was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by using the current coinmarketcap prices.
+
```
Comment (mhairimca) [inline Collateral-Omnibus-8.md:31]: Worth commenting more on IDIA, given the liquidity situation?

File: Collateral-Omnibus-8.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+This change has no implementations other than the aforementioned governor transactions
+
+## Security considerations
+
+Adding a collateral currency introduces a level of risk into the UMA Ecosystem.  This collateral type should be monitored to ensure that the proposed collateral continues to have value.
+
+Contract deployers considering using this collateral in an UMA contract should refer to the [guidelines on collateral type usage](https://docs.umaproject.org/uma-tokenholders/guidence-on-collateral-currency-addition) to ensure appropriate use.
+
+
```
Comment (mhairimca) [inline Collateral-Omnibus-8.md:44]: Again a specific comment about IDIA might be good?

File: Collateral-Omnibus-8.md (hunk 1 above)
Comment (inalittlewhile) [reply Collateral-Omnibus-8.md:27]: Verified the AQUA token address matches (https://etherscan.io/address/0xd34a24006b862f4e9936c506691539d6433ad297) the data provided. 

I can see an AQUA value of $.0006 with $61,929 in liquidity on https://v2.info.uniswap.org/token/0xd34a24006b862f4e9936c506691539d6433ad297

700,000 AQUA would be $420 at a value of $0.0006 each.

**Pull request merged**